pkg/box/fetch.go

```diff
@@ -65,6 +65,12 @@ func ApplyEnvOverrides(s *Config) {
 		s.AWS.DefaultRole = role
 	}
 
+	if a := os.Getenv("BOX_DOCKER_PUSH_IMAGE_REGISTRIES"); a != "" {
+		if pushRegs := strings.Split(a, " "); len(pushRegs) != 0 {
```

Comment:
```suggestion
		// The env var is space-separated since it's easiest to split in bash
		if pushRegs := strings.Split(a, " "); len(pushRegs) != 0 {
```